vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -22551,6 +22551,19 @@ unit:M2-PER-HA
   rdfs:label "square meters per hectare"@en-us ;
   rdfs:label "square metres per hectare"@en ;
 .
+unit:M2-PER-HA-YR
+  rdf:type qudt:Unit ;
+  qudt:conversionMultiplier 0.000000000003168808781402895 ;
+  qudt:conversionMultiplierSN 3.168808781402895e-12 ;
+  qudt:hasDimensionVector qkdv:A0E0L0I0M0H0T-1D1;
+  qudt:plainTextDescription "power of the SI base unit metre with the exponent 2 divided by the unit hectare and by the unit year";
+  qudt:hasQuantityKind quantitykind:Unknown ;
+  qudt:symbol "m²/(ha·yr)" ;
```

Comment:
This reminds me - we just adopted "a" instead of "yr" within qudt:symbol, but this is a note to @fkleedorfer about embedded occurrences of "yr" that were not caught in the last PR.

For your current PR, could you change all the "yr" to "a" in your submitted qudt:symbol triples?